tests/test_autonomy/test_cli/test_analyse/test_abci/test_specs.py

```diff
@@ -255,12 +253,6 @@ def test_failures(
             "Please provide path to specification file." in result.output
         ), result.output
 
-    @classmethod
```

Comment:
see above

Reply:
this is exactly one of the points i mentioned this morning I want to bring up in engineering sync you think people are aware of. They are not